docs/reference/commandline/build.md

```diff
@@ -31,7 +31,7 @@ parent = "smn_cli"
       --pull=false                    Always attempt to pull a newer version of the image
       -q, --quiet=false               Suppress the verbose output generated by the containers
       --rm=true                       Remove intermediate containers after a successful build
-      -t, --tag=""                    Repository name (and optionally a tag) for the image
+      -t, --tag=[]                    Repository names (and optionally with tags) to apply to the image
```

Comment:
@moxiegirl  The maximum length of each line in the `--help` command is limited to 90 characters.
See https://github.com/docker/docker/blob/master/integration-cli/docker_cli_help_test.go#L152